### ISSUE TITLE
Accessibility Issue: Small Arrow Slider Buttons Impair Usability

### ISSUE THREAD

**Issue by noobnarayan** [piyushgarg-dev/piyushgargdev-nextjs#392]
### Describe the bug

On the homepage, there are two slider components that exhibit a critical accessibility issue due to their excessively small arrow buttons. This issue impacts user interaction and navigation on the website.

**Location of the Sliders:**

The first slider is located immediately below the "Join Discord Server" button, and it is used for showcasing videos.
The second slider is situated within the Testimonial section.

**Issue Description:**

The arrow buttons that facilitate navigation within these sliders are significantly undersized, making it challenging for users, particularly those with mobility or visual impairments, to interact with them effectively. This not only violates basic accessibility standards but also results in a poor user experience.

First:
<img width="230" alt="image" src="https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/51719413/905e57ae-f57c-4231-b3e1-9b2129df62f6">

Second:

<img width="923" alt="image" src="https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/51719413/07ffb820-1004-490c-b8ed-749bf4e3d19d">

### To Reproduce

1. Open the homepage.
2. Locate the first slider beneath the "Join Discord Server" button.
3. Locate the second slider in testimonial section.
4. Attempt to use the arrow buttons for slider navigation.
5. Observe the difficulty in targeting and interacting with these buttons.

### Expected behavior

Ideally, the arrow buttons should be of an appropriate size, ensuring they are easily targetable and clickable for all users, including those who rely on assistive technologies.

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened
